compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -32,10 +32,10 @@ class CircleOptimizer final
   {
     enum Algorithm
     {
+      FuseBCQ,
       FuseInstanceNorm,
       ResolveCustomOpBatchMatMul,
-      QuantizeWithMinMax,
-      FuseBCQ,
+      QuantizeWithMinMax
```

Comment:
Is it possible to add `,` with `QuantizeWithMinMax,` ?
This way we don't have to change last line when adding as a last line